cfgov/core/templatetags/richtext.py

```diff
@@ -0,0 +1,22 @@
+from django import template
+
+
+register = template.Library()
+
+
+@register.filter
+def richtext_isempty(value):
+    """
+    Returns True if a value is None, an empty string, or empty paragraph tags
+
+    Working around known issue https://github.com/wagtail/wagtail/issues/3111.
+    Draftail RichTextBlocks passed in require accessing the `source` attribute
+    for the comparison.
+    """
+
+    blank_values = [None, '', '<p></p>']
+
+    return any([
+        value in blank_values,
+        hasattr(value, 'source') and value.source in blank_values,
+    ])
```

Comment:
Sorry to keep coming back to this, but it occurred to me that our rich text is pretty messy, and we might want to catch cases where the rich text just has spaces, such as `'<p>     </p>'` or `'  '`.

If so, this would handle them: 

```suggestion
    blank_values = [None, '', '<p></p>']

    return any([
        value.replace(' ', '') in blank_values,
        hasattr(value, 'source') and value.source.replace(' ', '') in blank_values,
    ])
```

Reply:
Good call!

Reply:
Sorry for the bum steer (can't call `replace` on None).

Not as clean, but I think this would do if if you want to catch spaces.

```python
    blank_values = ['', '<p></p>']

    return any([
        value is None or value.replace(' ', '') in blank_values,
        hasattr(value, 'source') and value.source is None or value.source.replace(' ', '') in blank_values,  # noqa: 501
    ])
```